gno.land/cmd/gnoland/start.go

```diff
@@ -151,17 +149,25 @@ func execStart(c *startCfg, args []string, io *commands.IO) error {
 		return fmt.Errorf("error in creating new app: %w", err)
 	}
 
-	cfg.LocalApp = gnoApp
+	// prune nothing is the archive node setting
+	// prune syncable is the default node setting. It keeps the lastest 100 transactions and everything 1000th tx
+	prune := store.NewPruningOptionsFromString(c.pruneStrategy)
+	pruningOpt := sdk.SetPruningOptions(prune)
+	gnoBaseApp := gnoApp.(*sdk.BaseApp)
+	pruningOpt(gnoBaseApp)
+
+	tmcfg.LocalApp = gnoBaseApp
+	tmcfg.P2P.PersistentPeers = c.persistentPeers
 
-	gnoNode, err := node.DefaultNewNode(cfg, logger)
+	gnoNode, err := node.DefaultNewNode(tmcfg, logger)
 	if err != nil {
 		return fmt.Errorf("error in creating node: %w", err)
 	}
 
-	fmt.Fprintln(io.Err, "Node created.")
+	fmt.Println("Node created.")
 
 	if c.skipStart {
```

Comment:
If we introduce a new "init" subcommand, we need to remove the "--skip-start" option from the "start" subcommand. This option was used to execute an initialization phase in a command before running on an initialized node.

For additional information, please refer to this PR: #1050